modules/cli/src/main/scala/scala/cli/commands/packaging/Spark.scala

```diff
@@ -23,4 +23,10 @@ object Spark {
 
   def sparkModules: Seq[AnyModule] =
     names.map(name => mod"org.apache.spark::spark-$name")
+
+  def hadoopModules: Seq[AnyModule] =
+    Seq(
+      // TODO Add more for Hadoop 2, maybe for 3 too
```

Comment:
NIT might be worth it to document in a ticket and leave a reference in this comment

Reply:
I'm not sure... Let's wait for feedback on those features in a first time.